masterlist.yaml

```
@@ -20995,7 +20995,11 @@ plugins:
 
   - name: 'CaranthirTowerReborn.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/4269/' ]
-    msg: [ *patch3rdParty_SRPC ]
+    msg:
+      - *patch3rdParty_SRPC 
```

There's an extra whitespace here:

```suggestion
      - *patch3rdParty_SRPC
```